lib/logstash/codecs/identity_map_codec.rb

```diff
@@ -52,14 +52,15 @@ def start
         while running? do
           sleep @interval
           break if !running?
+          break if @listener.nil?
           @listener.send(@method_symbol)
         end
       end
       self
     end
 
     def running?
-      @running.value
+      @running.true?
```

Comment:
is this necessary since running is a `AtomicBoolean` already?

Reply:
I asked myself why did the loop miss the `break if !running?` on line 54.
I have not dug into the AtomicBoolean code to see whether this directly helps but I feel that it is equally correct.